src/cbc_sdk/connection.py

```diff
@@ -530,13 +536,31 @@ def post_object(self, uri, body, **kwargs):
         Args:
             uri (str): The URI to send the POST request to.
             body (object): The data to be sent in the body of the POST request.
-            **kwargs:
+            **kwargs (dict): Additional arguments for the HTTP POST.
 
         Returns:
             object: The return data from the POST request.
         """
         return self.api_json_request("POST", uri, data=body, **kwargs)
 
+    def post_and_get_stream(self, uri, body, stream_output, **kwargs):
+        """
+        Send a POST request  to the specified URI and stream the results back into the given stream object.
+
+        Args:
+            uri (str): The URI to send the POST request to.
+            body (object): The data to be sent in the body of the POST request.
+            stream_output (RawIOBase): The output stream to write the data to.
+            **kwargs (dict): Additional arguments for the HTTP POST.
+
+        Returns:
+            object: The return data from the POST request.
+        """
+        with self.api_json_request("POST", uri, data=body, raw_result=True, stream=True, **kwargs) as resp:
```

Comment:
Remove `raw_result` from `api_json_request` and use `http_request` directly

Reply:
Done.